### PR TITLE
net-libs/libwebsockets: Bump to 4.2.0

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: net-libs/libwebsockets/libwebsockets-4.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		-DLWS_WITH_LIBEVENT=$(usex libevent)
+		-DLWS_WITH_LIBUV=$(usex libuv)
+		-DLWS_WITH_MBEDTLS=$(usex mbedtls)
+		-DLWS_WITH_PEER_LIMITS=$(usex peer-limits)
+		-DLWS_WITH_SERVER_STATUS=$(usex server-status)
+		-DLWS_WITH_SMTP=$(usex smtp)
+		-DLWS_WITH_SOCKS5=$(usex socks5)
+		-DLWS_WITH_SQLITE3=$(usex sqlite3)
+		-DLWS_WITH_SSL=$(usex ssl)
+		-DLWS_WITH_STATIC=$(usex static-libs)
```
Comment (SoapGentoo) [inline net-libs/libwebsockets/libwebsockets-4.2.0.ebuild:72]: do we really need this?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: net-libs/libwebsockets/libwebsockets-4.2.0.ebuild (hunk 1 above)
Comment (jsmolic) [reply net-libs/libwebsockets/libwebsockets-4.2.0.ebuild]: nope, I've removed it

**Comment by thesamesam** [comment]
Sorry, I had the REQUIRED_USE fix queued. Let me merge the rest.

**Comment by jsmolic** [comment]
> Sorry, I had the REQUIRED_USE fix queued already ([467bffc](https://github.com/gentoo/gentoo/commit/467bffc308bf969f4d4ba54f27069db0609a5daf) ) and missed this. Let me merge the rest.

No worries man, thank you for fixing it :+1: I've rebased now

**Pull request closed without merging**